includes/SVG.php

```diff
@@ -0,0 +1,390 @@
+<?php
+/**
+ * Class SVG.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+use SimpleXMLElement;
+use WP_Error;
+use Google\Web_Stories_Dependencies\enshrined\svgSanitize\Sanitizer;
+
+/**
+ * Class SVG
+ *
+ * @package Google\Web_Stories
+ */
+class SVG {
+	/**
+	 * File extenstion.
+	 *
+	 * @var string
+	 */
+	const EXT = 'svg';
+
+	/**
+	 * Mime type.
+	 *
+	 * @var string
+	 */
+	const MIME_TYPE = 'image/svg+xml';
+
+	/**
+	 * Array of SimpleXMLElements.
+	 *
+	 * @var array
```

Comment:
From what I've gathered it's an array of file contents.

```suggestion
	 * Cached list of SVG files and their contents.
	 *
	 * Speeds up access during the same request.
	 *
	 * @var string[]
```